config/crd/bases/atlas.mongodb.com_atlasprivateendpoints.yaml

```diff
@@ -106,8 +106,8 @@ spec:
                 - name
                 type: object
               externalProjectRef:
-                description: ExternalProject holds the Atlas project ID the user belongs
-                  to
```

Comment:
cc @helderjs have you find this typo in your change? The description name should match the actual field name